pages/heft_tasks/eslint.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: page
+title: '"eslint" task'
+navigation_source: docs_nav
+---
+
+This task invokes the [ESLint](https://eslint.org/) tool which reports errors about common coding problems.
+
+## When to use it
+
+ESLint fits together with several other tools as part of Rush Stack's strategy for code validation:
```

Comment:
It may be useful to expand on the point that this is our recommended approach, but it requires some manual setup for your own repo, and that we don't enforce that you use prettier or not use stylistic lint rules.

Reply:
Fixed